tools/perfcompare.ps1

```diff
@@ -7,21 +7,29 @@ param (
     [string]$DataFile1,
 
     [Parameter(Mandatory=$true)]
-    [string]$DataFile2
+    [string]$DataFile2,
+
+    [Parameter(Mandatory=$false)]
+    [string]$Metric = "pps"
 )
 
+Set-StrictMode -Version 'Latest'
+$ErrorActionPreference = 'Stop'
+
 function ImportDataset {
     param($File)
 
     $dataset = [ordered]@{}
-
-    $contents = Get-Content $File
-
-    foreach ($line in $contents) {
-        $array = $line.Split(",")
-        $scenarioName = $array[0]
-        $scenarioData = $array[3..$array.Count]
-        $dataset.Add($scenarioName, $scenarioData)
+    $contents = Get-Content -Raw $File | ConvertFrom-Json
+
+    foreach ($data in $contents) {
+        $scenarioName = $data.ScenarioName
+        $scenarioData = ($data.Metrics | Where-Object { $_.Name -eq $Metric }).Value
```

Comment:
Is it possible that `Metrics` will contain several items with the same Name? I what happens here in that case (does it takes the first or fail?)

Reply:
My expectation is each metric must have a unique `Name` field. Of course, there is no formal schema yet. In terms of error handling, I think undefined behavior is fine for now.